avionics/envs/x86/include/sensors/barometer.h

```diff
@@ -21,5 +22,6 @@ class Barometer : public SensorBase<2> {
     SensorStatus getStatus() { return SensorStatus::NOMINAL; }
 
   private:
-    DataSpoof<2> dat{BAROMETER_DATA, data_};
```

Comment:
`BAROMETER_DATA` doesn't seem to have any use anymore; if the definition hasn't been removed make sure to do so